--- trusted-firmware-a-2.5/plat/rockchip/rk3399/drivers/dram/dram.h.orig	2021-05-14 17:56:37.000000000 +0200
+++ trusted-firmware-a-2.5/plat/rockchip/rk3399/drivers/dram/dram.h	2021-05-18 13:59:59.413453148 +0200
@@ -149,7 +149,7 @@
 	uint32_t rx_cal_dqs[2][4];
 };
 
-extern __sramdata struct rk3399_sdram_params sdram_config;
+extern __pmusramdata struct rk3399_sdram_params sdram_config;
 
 void dram_init(void);
 
